packages/pcd/zk-eddsa-frog-pcd/package.json

```diff
@@ -30,15 +30,16 @@
   ],
   "scripts": {
     "lint": "eslint \"**/*.ts{,x}\"",
-    "build": "yarn build:browser && yarn build:node && yarn build:types",
+    "build": "yarn build:ts && yarn build:types",
+    "build:ts": "yarn build:browser && yarn build:node",
     "build:browser": "tsup src/index.ts --platform=browser --out-dir=./dist/browser --format esm --clean",
     "build:node": "tsup src/index.ts --platform=node --out-dir=./dist/node --format cjs,esm --clean",
-    "build:types": "rm -rf dist/types && tsc --emitDeclarationOnly --outDir dist/types",
+    "build:types": "rm -f tsconfig.build.tsbuildinfo && tsc -p tsconfig.build.json --emitDeclarationOnly",
     "typecheck": "yarn tsc --noEmit",
     "test": "ts-mocha --config ../../../.mocharc.js --exit test/**/*.spec.ts",
-    "prepublishOnly": "yarn build",
+    "prepublishOnly": "yarn build && mkdir -p artifacts && cp src/artifacts/* ./artifacts/*",
```

Comment:
this is a bit confusing to me. a little sketchy that `prepublishOnly` does more than just `build`. perhaps this `cp` could be moved to the `build` script?

Reply:
Resolved in the new PR.